### ISSUE TITLE
clientv3: go get go.etcd.io/etcd/clientv3 always fail.

### ISSUE THREAD

**Issue by cupen** [etcd-io/etcd#12276]
* go 1.15

create a go module.
```
mkdir test && cd test && go mod init test
```

go get go.etcd.io/etcd/clientv3
```
$ go get go.etcd.io/etcd/clientv3 
go: found go.etcd.io/etcd/clientv3 in go.etcd.io/etcd v3.3.25+incompatible
go: finding module for package github.com/coreos/etcd/auth/authpb
go: finding module for package github.com/google/uuid
go: finding module for package google.golang.org/grpc/keepalive
go: finding module for package github.com/coreos/etcd/clientv3/credentials
go: finding module for package google.golang.org/grpc/status
go: finding module for package google.golang.org/grpc
go: finding module for package github.com/coreos/pkg/capnslog
go: finding module for package github.com/coreos/etcd/pkg/types
go: finding module for package github.com/coreos/etcd/clientv3/balancer
go: finding module for package go.uber.org/zap
go: finding module for package google.golang.org/grpc/grpclog
go: finding module for package github.com/coreos/etcd/clientv3/balancer/picker
go: finding module for package github.com/coreos/etcd/etcdserver/api/v3rpc/rpctypes
go: finding module for package github.com/coreos/etcd/clientv3/balancer/resolver/endpoint
go: finding module for package github.com/coreos/etcd/etcdserver/etcdserverpb
go: finding module for package github.com/coreos/etcd/version
go: finding module for package github.com/coreos/etcd/mvcc/mvccpb
go: finding module for package google.golang.org/grpc/codes
go: finding module for package google.golang.org/grpc/metadata
go: finding module for package github.com/coreos/etcd/pkg/logutil
go: finding module for package google.golang.org/grpc/credentials
go: found github.com/coreos/etcd/auth/authpb in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/clientv3/balancer in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/clientv3/balancer/picker in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/clientv3/balancer/resolver/endpoint in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/clientv3/credentials in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/etcdserver/api/v3rpc/rpctypes in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/etcdserver/etcdserverpb in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/mvcc/mvccpb in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/pkg/logutil in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/pkg/types in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/etcd/version in github.com/coreos/etcd v3.3.25+incompatible
go: found github.com/coreos/pkg/capnslog in github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
go: found github.com/google/uuid in github.com/google/uuid v1.1.2
go: found go.uber.org/zap in go.uber.org/zap v1.16.0
go: found google.golang.org/grpc in google.golang.org/grpc v1.31.1
go: found google.golang.org/grpc/codes in google.golang.org/grpc v1.31.1
go: found google.golang.org/grpc/credentials in google.golang.org/grpc v1.31.1
go: found google.golang.org/grpc/grpclog in google.golang.org/grpc v1.31.1
go: found google.golang.org/grpc/keepalive in google.golang.org/grpc v1.31.1
go: found google.golang.org/grpc/metadata in google.golang.org/grpc v1.31.1
go: found google.golang.org/grpc/status in google.golang.org/grpc v1.31.1
go: finding module for package github.com/gogo/protobuf/gogoproto
go: finding module for package github.com/coreos/go-systemd/journal
go: finding module for package github.com/coreos/go-semver/semver
go: found github.com/gogo/protobuf/gogoproto in github.com/gogo/protobuf v1.3.1
go: found github.com/coreos/go-systemd/journal in github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
go: found github.com/coreos/go-semver/semver in github.com/coreos/go-semver v0.3.0
# github.com/coreos/etcd/clientv3/balancer/resolver/endpoint
../../../go/pkg/mod/github.com/coreos/etcd@v3.3.25+incompatible/clientv3/balancer/resolver/endpoint/endpoint.go:114:78: undefined: resolver.BuildOption
../../../go/pkg/mod/github.com/coreos/etcd@v3.3.25+incompatible/clientv3/balancer/resolver/endpoint/endpoint.go:182:31: undefined: resolver.ResolveNowOption
# github.com/coreos/etcd/clientv3/balancer/picker
../../../go/pkg/mod/github.com/coreos/etcd@v3.3.25+incompatible/clientv3/balancer/picker/err.go:37:44: undefined: balancer.PickOptions
../../../go/pkg/mod/github.com/coreos/etcd@v3.3.25+incompatible/clientv3/balancer/picker/roundrobin_balanced.go:55:54: undefined: balancer.PickOptions
```

It seems that v3.4 will fix it, but it's not found.
v3.4.13
```
$ go get go.etcd.io/etcd/clientv3@v3.4.13
go get go.etcd.io/etcd/clientv3@v3.4.13: go.etcd.io/etcd@v3.4.13: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v3
```

v3.4.2.
```
$ go get go.etcd.io/etcd/clientv3@v3.4.2
go get go.etcd.io/etcd/clientv3@v3.4.2: go.etcd.io/etcd@v3.4.2: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v3
```

**Comment by cupen** [comment]
Maybe you can release tag v3.4.13rc1?  rc is `Release Candidate`.

**Comment by cupen** [comment]
It seems that would be fully fixed at [milestone-v3.5].  #11154
But, there is too many job to do like #12124

[milestone-v3.5]: https://github.com/etcd-io/etcd/milestone/37
